From 1abe23a02e86f02f94ad155cc3ada18147d3dc3f Mon Sep 17 00:00:00 2001
From: Andy Shevchenko <andriy.shevchenko@linux.intel.com>
Date: Thu, 26 Mar 2020 14:21:15 +0200
Subject: [PATCH 75/89] platform/x86: surface3_power: Reformat GUID assignment

For better readability reformat GUID assignment.
While here, add the comment how this GUID looks in a string representation.

Signed-off-by: Andy Shevchenko <andriy.shevchenko@linux.intel.com>
---
 drivers/platform/x86/surface3_power.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/drivers/platform/x86/surface3_power.c b/drivers/platform/x86/surface3_power.c
index 24010e0696a0..692ba845a1b4 100644
--- a/drivers/platform/x86/surface3_power.c
+++ b/drivers/platform/x86/surface3_power.c
@@ -147,9 +147,10 @@ struct gsb_buffer {
 
 #define MSHW0011_EV_2_5_MASK		GENMASK(8, 0)
 
+/* 3f99e367-6220-4955-8b0f-06ef2ae79412 */
 static const guid_t mshw0011_guid =
-	GUID_INIT(0x3F99E367, 0x6220, 0x4955, 0x8B, 0x0F, 0x06, 0xEF, 0x2A,
-		  0xE7, 0x94, 0x12);
+	GUID_INIT(0x3F99E367, 0x6220, 0x4955, 0x8B, 0x0F, 0x06, 0xEF,
+		  0x2A, 0xE7, 0x94, 0x12);
 
 static int
 mshw0011_notify(struct mshw0011_data *cdata, u8 arg1, u8 arg2,
-- 
2.30.2

